pipeline/run_benchmark.sh

```diff
@@ -231,18 +240,27 @@ EOT
                      sed "s/ACCINDEX/-account_index $index/" $LOGDIR/$cmd/$cmd.json > $LOGDIR/$cmd/$cmd-$ip.json
                   fi
                fi
+               if [ "${configs[benchmark.bls]}" == "true" ]; then
+                  bls=${blskey[$start_index]}
+                  sed -i "s/BLSKEY/$bls/" $LOGDIR/$cmd/$cmd-$ip.json
+                  ./node_ssh.sh -d $LOGDIR ec2-user@$ip "aws s3 cp s3://${configs[bls.bucket]}/${configs[bls.folder]}/$bls $bls"
```

Comment:
These SSH calls are executed in sequence and will take minutes of time.  Can we background them and `wait` for them right before we execute `$cmd_file`?